main/webapp/modules/core/scripts/reconciliation/recon-manager.js

```diff
@@ -63,6 +63,14 @@ ReconciliationManager.registerService = function(service) {
 };
 
 ReconciliationManager.registerStandardService = function(url, f, silent) {
+
+  if (ReconciliationManager._urlMap.hasOwnProperty(url)) {
+    if (!silent) {
+      alert($.i18n('core-recon/url-already-registered'));
+    }
+    if (f) { f(url); }
```

Comment:
We probably still need your return statement here, so that the rest of the function is not executed.